### PR TITLE
#1414 Update key individuals

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/eto/registration/pages/KeyIndividuals.tsx [hunk 1]
```diff
@@ -59,142 +60,162 @@ const getBlankMember = () => ({
...
+);
 
 class KeyIndividualsGroup extends React.Component<IKeyIndividualsGroup> {
   static contextTypes = {
     formik: PropTypes.object,
   };
 
-  componentWillMount(): void {
-    const { setFieldValue, values } = this.context.formik as FormikProps<any>;
+  isEmpty(): boolean {
```
Comment (desfero) [inline app/components/eto/registration/pages/KeyIndividuals.tsx]: @trival @Moejoe90 I'm wondering whether it's better to use getter here over function?

Comment (Moejoe90) [reply app/components/eto/registration/pages/KeyIndividuals.tsx]: From my side i have no strong opinions regarding this

File: app/components/eto/shared/EtoPublicComponent.utils.ts [hunk 2]
```diff
@@ -13,3 +15,16 @@ export const selectActiveCarouselTab = (elements: any[]): number => {
 
   return 0;
 };
+
+export const areThereIndividuals = (
```
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/eto/shared/EtoPublicComponent.utils.ts]: no, my mistake.`eto/utils.ts` is a duplicate

File: app/components/eto/registration/pages/KeyIndividuals.tsx [hunk 3]
```diff
@@ -40,16 +42,15 @@ type IProps = IStateProps & IDispatchProps & FormikProps<TPartialCompanyEtoData>
...
-  isFirst: boolean;
+  canRemove: boolean;
   index: number;
   groupFieldName: string;
 }
 
 interface IKeyIndividualsGroup {
   name: string;
   title: TTranslatedString;
+  validationSchema: Schema<any>;
```
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/eto/registration/pages/KeyIndividuals.tsx]: fixed

**Pull request merged**